### PR TITLE
Add lgtm.co config

### PR DESCRIPTION
**Pull request by rsalevsky**

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
+1

File: .lgtm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+approvals = 2
+pattern = "(?i):\\+1:|\\+1"
```
Comment (nicolasbock) [inline .lgtm]: Also include `(?i)lgtm`?

Comment (rsalevsky) [inline .lgtm]: Nop, we decided that LGTM does not count as +1. It's more like looks ok but I have no real idea what happens in detail to decide.

Comment (nicolasbock) [inline .lgtm]: Ok, sounds good, just wanted to make since I had seen this before in comments.

+1

**Pull request merged**